Sources/Logging/Logging.swift

```diff
@@ -96,25 +96,46 @@ public struct Logger {
     }
 }
 
-// This is the logging system itself, it's mostly used to obtain loggers and to set the type of the `LogHandler`
-// implementation.
-public enum Logging {
```

Comment:
@hartbit how would you feel about instead renaming `Logging` to `LoggingSystem`? Previously we went for `Logging` because `Logging.make(...)` was such an important function. But after your PR, `Logging.make(...)` would be replaced by `Logger(...)`. So the importance of `Logging` itself is vastly reduced.

I feel like we should rename it to `LoggingSystem` and keep the `bootstrap` and `make` (which maybe should be `makeLogger`) methods. In most programs, the type `LoggingSystem` will only be used once: to bootstrap the logging system with

    LoggingSystem.bootstrap(...)

and I think that reads well.

Reply:
Sounds like a nice compromise